Procfile

```diff
@@ -1,2 +1,3 @@
 web:         bundle exec passenger start -p $PORT --max-pool-size $PASSENGER_MAX_POOL_SIZE
 worker:      QUEUES=default,paperclip,mailers bundle exec rake jobs:work
+image_reprocess_worker: QUEUE=image_reprocess bundle exec rake jobs:work
```

Comment:
Is this necessary? Why not just add to the normal worker queue with low enough priority? (11)